dca/methods_comparison.py

```diff
@@ -694,8 +699,22 @@ def fit(self, X):
         dX = np.concatenate([np.diff(Xi, axis=0) for Xi in X_red], axis=0)
         X_prestate = np.concatenate([Xi[:-1] for Xi in X_red], axis=0)
         M_skew = self._fit_skew(X_prestate, dX)
+
         self.eigen_vals_, self.eigen_vecs_ = self._get_jpcs(M_skew)
 
+        self.proj_vectors_ = []
+        for i in range(len(self.eigen_vecs_) // 2):
+            v1 = self.eigen_vecs_[2 * i]
+            v2 = self.eigen_vecs_[2 * i + 1]
+            real_v1 = np.real(v1 + v2)
+            real_v1 /= np.linalg.norm(real_v1)
+
+            real_v2 = np.imag(v1 - v2)
+            real_v2 /= np.linalg.norm(real_v2)
+            # remove 0j
+            self.proj_vectors_.append(real_v1)
+            self.proj_vectors_.append(real_v2)
+
```

Comment:
I would convert `self.proj_vectors_` to a np array here rather than having to do it during transform.

Reply:
Done!